dist/src/main/resources/standalone.xsl

```diff
@@ -216,6 +216,21 @@
     <xsl:call-template name="loggers"/>
   </xsl:template>
 
+  <!-- Enable gzip compression when serving javascript and ccs -->
+  <xsl:template match="undertow:subsystem/undertow:server/undertow:host">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|@*"/>
+      <filter-ref name="gzipFilter" predicate="path-suffix['.css'] or path-suffix['.js']"/>
```

Comment:
We can add those additional path-suffix values, and anything else we deem necessary.